libs/styles/src/lib/scss/components/_datepicker.scss

```diff
@@ -0,0 +1,141 @@
+$datepicker-border-color: #9799A4;
+$datepicker-border-radius: 4px;
+$datepicker-font-size: 16px;
+$datepicker-width: 125px;
+$datepicker-height: 40px;
+$datepicker-color: #34394C;
+
+@mixin cvi-datepicker($base: &) {
+  --comment: 'A datepicker';
+
+  @at-root {
+    #{$base}__input {
+      &-container {
+        border: 1px solid $datepicker-border-color;
+        border-radius: $datepicker-border-radius;
+        display: flex;
+        align-items: center;
+        width: $datepicker-width;
+        height: $datepicker-height;
+
+        &-focused {
+          border-color: get-color(sapphire-blue-10);
+        }
+      }
+
+      &-field {
+        font-size: $datepicker-font-size;
+        border-left: 1px;
+        border-radius: $datepicker-border-radius;
+        padding: 8px;
+        width: 96px;
+
+        &:focus {
+          outline: none;
+        }
+      }
+
+      &-icon-wrapper {
+        width: 29px;
+        padding-top: 6px;
+      }
+
+      #{$base}--is-disabled & {
+        background-color: get-color(black-coral-0);
+        border-color: get-color(black-coral-5);
+      }
+    }
+
+    #{$base}__calendar {
+      box-sizing: border-box;
+      position: absolute;
+      background: get-color(white);
```

Comment:
It's better to avoid some shorthand properties with a big amount of possible values because they can cause hard to track bugs.